energy-integration/governance-v2/src/views.rs

```diff
@@ -100,6 +104,10 @@ pub trait ViewsModule:
         proposal_id >= 1 && proposal_id <= self.proposals().len()
     }
 
+    fn proposal_reached_min_fees(&self, proposal_id: ProposalId) -> bool {
+        self.proposals().get(proposal_id).fees.total_amount >= self.min_fee_for_propose().get()
```

Comment:
It'd a bit cleaner if each comparison member is evaluated separately
```suggestion
        let accumulated_fees = self.proposals().get(proposal_id).fees.total_amount;
        let min_fees = self.min_fee_for_propose().get();
        accumulated_fees >= min_fees
```